Header header
int64 path_len
float64[] speed
float64[] steering
float64[] duration
